launch_checklist.module

```diff
@@ -117,5 +138,572 @@ function launch_checklist_checklistapi_checklist_items() {
         '#description' => t('Ensure Spam prevention measures are in place.'),
       ],
     ],
+
+    // Section 4 SEO.
+    '04_seo' => [
+      '#title' => t('SEO'),
+      '#description' => t('<p>SEO settings for the Drupal site.</p>'),
+
+      'analytics' => [
+        '#title' => t('Analytics'),
+        '#description' => t('Verify Google Analytics or similar is installed and correct access has been given'),
+        'handbook_page' => [
+          '#text' => t('Google Analytics module'),
+          '#url' => $google_analytics_route,
+        ],
+      ],
+      'xml_sitemap' => [
```

Comment:
Drupal 8+ has two great options for creating XML sitemaps, xml_sitemap and simple_sitemap.  

Could `handbook_page` accept 2 links? (or be altered, or be added twice?)

Same could be said for analytics, we could also like Google Tag Manager, Piwik, etc...